ios/brave-ios/Sources/BraveVPN/Components/Region/BraveVPNRegionListView.swift

```diff
@@ -42,37 +42,37 @@ public struct BraveVPNRegionListView: View {
   }
 
   public var body: some View {
-    ZStack {
-      VStack {
-        List {
-          Section(
-            footer: Text(Strings.VPN.serverRegionAutoSelectDescription)
-              .font(.footnote)
-              .foregroundStyle(Color(braveSystemName: .textSecondary))
-          ) {
-            automaticRegionToggle
-          }
+    VStack {
```

Comment:
`VStack` here is unneeded

Reply:
Fix at https://github.com/brave/brave-core/pull/24198/commits/e62c725010a6192dd0df24c1c91e3d07eb3f52ee